app/helpers/application_helper.rb

```diff
@@ -24,6 +24,10 @@ def bootstrap_class_for_flash(flash_type)
   end
 
   def display_date(date_time)
+    unless date_time.present?
+      date_time
+
+    end
     if date_time.is_a? String
       DateTime.parse(date_time).strftime(STANDARD_DATE_DISPLAY_FORMAT)
     else
```

Comment:
Can you explain the use of & in `date_time&.strftime()`? I don't understand it & Google didn't help

Reply:
`&.` will only call the function `strftime()` if `date_time` is not `nil`